components/Chips/examples/ChipsShapingExampleViewController.m

```diff
@@ -43,6 +44,13 @@ - (id)init {
   return self;
 }
 
+- (MDCContainerScheme *)containerScheme {
+  MDCContainerScheme *scheme = [[MDCContainerScheme alloc] init];
+  scheme.colorScheme = self.colorScheme;
+  scheme.typographyScheme = self.typographyScheme;
```

Comment:
Would you please add the shape scheme either here or as a follow-up?

Reply:
done